tests/migration_test.go

```diff
@@ -901,16 +902,27 @@ var _ = Describe("[rfe_id:393][crit:high[vendor:cnv-qe@redhat.com][level:system]
 				}
 				handler, err := kubecli.NewVirtHandlerClient(virtClient).ForNode(vmi.Status.MigrationState.TargetNode).Pod()
 				Expect(err).ToNot(HaveOccurred())
-				// The port-forwarder tears down after each check, but may be too slow, so use different ports on fast checks
-				for port, _ := range vmi.Status.MigrationState.TargetDirectMigrationNodePorts {
-					func() {
+
+				var wg sync.WaitGroup
+				wg.Add(len(vmi.Status.MigrationState.TargetDirectMigrationNodePorts))
```

Comment:
I am probably missing something but here all the funcs (and thus the `wg.Done()`) are executed sequentially since no go routines are involved, so is the wait group necessary?

Or maybe the funcs were meant to be spawned as goroutines?

Reply:
`tests.ForwardPorts` uses a go routine

Reply:
uhm yes, but the wg.Done is invoked on the main go routine I guess?

Reply:
Now there is. @fedepaol thanks. There should have been a `go ...`